packaging/windows/get-convert-licenses.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-3.0-or-later
+
```

Comment:
We should be using `set -e` here, since any part of this failing causes the whole thing to fail.